src/ConfiguresApplication.php

```diff
@@ -17,18 +17,20 @@ trait ConfiguresApplication
 {
     /**
      * Create the '.env' from '.env.example'.
-     *
-     * @param $reset
      */
-    public function createEnvironmentFile($reset)
+    public function createEnvironmentFile()
```

Comment:
This `$reset` variable wasn't actually being used correctly and would always return `false` I believe. It didn't seem like `$reset` was being set and I think it's because to retrieve it properly you need to use the `options()` method. Fixed it up and tested it out and seems to be working as intended now!